andela/ah-bird-box
# #164099801 Fixes Cloudinary image URL

#### What does this PR do?

This PR fixes the error on the image Url in Articles creation.

#### Description of Task to be completed?

Add a cloudinary field for image and a characterfield for url

#### How should this be manually tested?

Make the following requests on an API tester i.e. postman:

* the URL `http://127.0.0.1:8000/api/articles/` to create an article. or the URL `http://127.0.0.1:8000/api/articles/<str: slug>/` to edit an existing article.

Make sure to add an image to the request.

#### Any background context you want to provide?

The image must be uploaded to Cloudinary before receiving the image url.

#### What are the relevant pivotal tracker stories?

[#164099801](https://www.pivotaltracker.com/story/show/164099801)

#### Screenshots (if appropriate):

<img width="969" alt="screen shot 2019-02-22 at 10 46 42" src="https://user-images.githubusercontent.com/5156758/53227421-33df3480-368f-11e9-8238-4f0ddb766d9e.png">

<img width="963" alt="screen shot 2019-02-22 at 10 54 03" src="https://user-images.githubusercontent.com/5156758/53227750-3a21e080-3690-11e9-851f-b486ffefedea.png">


#### Questions:

None
